### ISSUE TITLE
is it really a3c implementation? not just actor critic?

### ISSUE THREAD

**Issue by sevity** [MorvanZhou/pytorch-A3C#8]
where is multi-step loss function and entropy loss function?

**Comment by KGBUSH** [comment]
For entropy exploration, you can check continuous_A3C.py in loss_func function

**Comment by KGBUSH** [comment]
> where is multi-step loss function and entropy loss function?

For entropy exploration, you can check continuous_A3C.py in loss_func function

**Comment by jakkarn** [comment]
This should be the if-condition that determines when N actions have been taken (or terminated) in the `Worker`:
https://github.com/MorvanZhou/pytorch-A3C/blob/5ab27abee2c3ac3ca921ac393bfcbda4e0a91745/discrete_A3C.py#L93

N-step loss is then computed in `push_and_pull` in this loop:
https://github.com/MorvanZhou/pytorch-A3C/blob/5ab27abee2c3ac3ca921ac393bfcbda4e0a91745/utils.py#L29
